apps/api/v2/src/modules/billing/billing.module.ts

```diff
@@ -33,6 +36,9 @@ import { Module } from "@nestjs/common";
     BillingRepository,
     BillingProcessor,
     ManagedOrganizationsBillingService,
+    UsersRepository,
```

Comment:
You see under `imports` above we import other modules, so that whatever they `expose` this module has access to.

We are already importing `UsersModule` which has `UsersRepository` in the `exports` section (see apps/api/v2/src/modules/users/users.module.ts), so you can remove `UsersRepository` from the `providers` in this file.